docs/references/protocol/transactions/types/oracledelete.md

```diff
@@ -29,8 +27,7 @@ _(Added by the [PriceOracle amendment][].)_
 
```

Comment:
We should add a link to the common transaction fields.

{% raw-partial file="/docs/_snippets/tx-fields-intro.md" /%}

Need to remove the existing "OracleDelete Fields" heading though as the above adds it automatically.